types/src/toGamut.d.ts

```diff
@@ -10,6 +10,13 @@ declare function toGamut (
 	options?: {
 		method?: string | undefined;
 		space?: string | ColorSpace | undefined;
+		deltaEMethod?: string | undefined;
+		jnd?: number | undefined;
+		blackWhiteClamp?: {
+			channel: string;
```

Comment:
It seems like this could be a `Ref` (imported from `types/src/space.d.ts`) instead of limited to a `string`?

Reply:
The current intention was a string. I'm not quite sure what you mean by Ref though.

Reply:
It's not a big deal, but from the implementation it looks like `blackWhiteClamp.channel` is only used to pass to `ColorSpace.resolveCoord`. [That function](https://github.com/color-js/color.js/blob/main/types/src/space.d.ts#L65) can accept a [Ref](https://github.com/color-js/color.js/blob/main/types/src/space.d.ts#L43-L46), which can be a string or array or object -- so I think any of those would also be acceptable here?

Reply:
I guess? When I ran the type linting, it complained about `Ref`. I'm happy to allow it if it doesn't break anything and I can get the type checker to not yell at me 🙃.

Reply:
My guess is it was complaining because it doesn't exist in that file, and needs to be imported.

Reply:
I think I fixed it.